src/Components/Navbar/Components/SearchModal/SearchModal.scss

```
@@ -0,0 +1,138 @@
+@import '../../../../Styles/common.scss';
+
+.search-modal {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  z-index: 500;
+
+  .modal-top {
+    height: 35%;
+    background: #ffffff;
+
+    .modal-top-header {
+      @extend %flexAlign;
+      justify-content: space-between;
+      height: 20%;
+      padding: 0 30px;
+
+      .modal-logo {
+        height: 45px;
+        img { height: 100%; }
+        &:hover {
+          opacity: 0.6;
+        }
+      }
```

```suggestion
      .modal-logo {
        height: 45px;
        
        img {
          height: 100%;
        }
        
        &:hover {
          opacity: 0.6;
        }
      }
```
다른 부분들과 굳이 규칙을 다르게 하여 작성할 필요 없을 것 같습니다. 가독성을 위해 일관된 규칙으로 작성해주세요.